pktgen_s/server/pktgen_server.c

```diff
@@ -0,0 +1,331 @@
+/*
+ * pktgen_server.c
+ * 	
+ * Runs as a daemon and listens to jobs from pktgen_scheduler.py
+ *
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <errno.h>
+#include <netdb.h>
+#include <netinet/in.h>
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <sys/stat.h>
+#include <signal.h>
+#include <sys/mman.h>
+#include <fcntl.h>
+#include <syslog.h>
+#include <sys/wait.h>
+#include <stdint.h>
+
+#include "job.pb-c.h"
+#include "status.pb-c.h"
+
+#define PORT "1729"
+#define SCHEDULER_IP "127.0.0.1"
+#define SCHEDULER_PORT "1800"
+#define BUFSIZE 8192
+#define BACKLOG 25
+
+static void 
+setup_daemon()
+{
+
+	pid_t pid;
+
+	// fork off parent process
```

Comment:
No need for all of these single line comments.

Most of the code here explains itself. Really only need comments to shed light on non-obvious/subtle stuff